src/components/Navigation/TopBar/Help/ReleaseNotesDialog/ReleasePosts/ReleasePost.styles.ts

```diff
@@ -0,0 +1,112 @@
+import { Button as EDSbutton, Typography } from '@equinor/eds-core-react';
+import { tokens } from '@equinor/eds-tokens';
+
+import styled from 'styled-components';
+
+const { spacings, shape, elevation } = tokens;
+
+const Container = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding: ${spacings.comfortable.xx_large};
+  background-color: #fff;
+  box-shadow: ${elevation.raised};
+  border-radius: ${shape.corners.borderRadius};
+  overflow: hidden;
+  width: calc(100% - ${spacings.comfortable.xx_large} * 2);
+`;
+
+const TopContainer = styled.div`
+  display: grid;
+  grid-template-columns: auto 1fr;
+  justify-items: end;
+  padding-top: 0;
+  padding-bottom: ${spacings.comfortable.medium_small};
+`;
+
+const HeadingContainer = styled.div`
+  display: flex;
+  flex-direction: row;
```

Comment:
Also something you don't need to change, but 'flex-direction: row' isn't needed since that is the default direction

Reply:
Removed